docs/howto_3_CustomizedAdvisor.md

```diff
@@ -0,0 +1,39 @@
+# **How To** - Customize Your Own Advisor
+
+*Advisor targets the scenario that the automl algorithm wants the methods of both tuner and assessor. Advisor is similar to tuner on that it receives trial configuration request, final results, and generate trial configurations. Also, it is similar to assessor on that it receives intermediate results, trial's end state, and could send trial kill command. Note that, if you use Advisor, tuner and assessor are not allowed to be used at the same time.*
+
+So, if user want to implement a customized Advisor, she/he only need to:
+
+1) Define an Advisor inheriting from the MsgDispatcherBase class
+2) Implement the handle_xxx methods
```

Comment:
handle_xxx methods from MsgDispathcherBase?

Reply:
ok, I can list all the methods.

Reply:
fixed, thx